src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,52 @@
+.nav {
+  margin-right: 50px;
+
```

Comment:
You should not add external margins for the separate blocks.

Reply:
agree, you can't set margins to BEM block